core/src/main/java/org/fao/geonet/kernel/metadata/DefaultStatusActions.java

```diff
@@ -84,7 +85,7 @@ public void init(ServiceContext context) throws Exception {
         siteName = sm.getSiteName();
         host = sm.getValue(Settings.SYSTEM_FEEDBACK_MAILSERVER_HOST);
         port = sm.getValue(Settings.SYSTEM_FEEDBACK_MAILSERVER_PORT);
-        from = sm.getValue(Settings.SYSTEM_FEEDBACK_EMAIL);
+        from = sm.getValue(SYSTEM_FEEDBACK_EMAIL);
```

Comment:
Is this change fine removing the `Settings.` prefix?

Reply:
It is only a matter of type of import 
```
import static org.fao.geonet.kernel.setting.Settings.SYSTEM_FEEDBACK_EMAIL;
```
Do you want to change that ? that is some intellij autocomplete feature.

Reply:
Done some cosmetic on import https://github.com/geonetwork/core-geonetwork/pull/3209/commits/e44338a03346d37eb276c5a2c89b509f16bcc26f they all look the same and only one about SMTP config is now used in that class.